CONTRIBUTING.md

```diff
@@ -0,0 +1,204 @@
+# Contribution Guidelines
+
+Thanks for checking out our Polkadot Runtime Implementation! We're excited to hear and learn from you.
+
+We've put together the following guidelines to help you figure out where you can best be helpful. The Web3 foundation has a comprehensive collection of [Polkadot Resources](https://github.com/w3f/web3/blob/537a2518c24e96b05ceadd9f31348669e72b8841/docs/layer_1/platforms/polkadot.md) for both part-time and core contributors to the project in order to get up to speed.
+
+Additionally, the [Polkadot Specification Doc](https://github.com/w3f/polkadot-spec/blob/master/spec.md) serves source of truth for all things related to our implementation, however it is currently in its final draft status so things may be subject to change.
+
+Feel free to fork our repo and start creating PR’s after assigning yourself to an issue of interest.
+
+## Contribution Steps
+
+**1. Set up Go-pre following the instructions in README.md.**
+
+**2. Fork the Go-pre repo.**
+
+Sign in to your Github account or create a new account if you do not have one already. Then navigate your browser to [https://github.com/ChainSafeSystems/go-pre](https://github.com/ChainSafeSystems/go-pre). In the upper right hand corner of the page, click “fork”. This will create a copy of the Go-pre repo in your account.
+
+**3. Create a local clone of Go-pre.**
+
+```
+$ mkdir -p $GOPATH/src/github.com/ChainSafeSystems
```

Comment:
@dutterbutter I believe with new go 1.11 modules, the project is not dependent on GOPATH anymore.